triaging.rst

```diff
@@ -375,6 +375,13 @@ Various informational flags about the issue. Multiple values are possible.
 | easy          | Fixing the issue should not take longer than a day for     |
 |               | someone new to contributing to Python to solve.            |
 +---------------+------------------------------------------------------------+
+| easy (C)      | Fixing the issue should not take longer than a day for     |
+|               | someone new contributing to Python, focused on C/C++.      |
```

Comment:
We just use C, so the mention of C++ here is out of place.

Reply:
Done.